Directory.Packages.props

```diff
@@ -25,7 +25,7 @@
     <PackageVersion Include="Azure.Storage.Blobs" Version="12.23.0" />
     <PackageVersion Include="Azure.Storage.Queues" Version="12.21.0" />
     <PackageVersion Include="Microsoft.Azure.AppConfiguration.AspNetCore" Version="8.0.0" />
-    <PackageVersion Include="Microsoft.Azure.Cosmos" Version="3.45.0" />
+    <PackageVersion Include="Microsoft.Azure.Cosmos" Version="3.45.2" />
```

Comment:
Why not 3.46.1 ?

I have this change in my PR and it required some specific actions but how did it pick this?

Reply:
This was the minimum needed to use the 9.0.1 of Microsoft.EntityFrameworkCore.Cosmos. 3.46 adds the new Newtonsoft.Json issues that I didn't want to tackle with this PR. We can update it past 3.46 in a separate PR.